app/helpers/application_helper.rb

```diff
@@ -310,3 +310,27 @@ def legacy_javascript_tag
   end
 end
 # rubocop:enable Metrics/ModuleLength
+
+# error_messages_for method was deprecated, however lots of the tests depend on the message format it
```

Comment:
Does this exist only for tests?

Reply:
No, it is replacing a method that has been deprecated and no longer worked. It is in use in the app now to display errors.